typed-protocols/src/Network/TypedProtocol/Driver.hs

```diff
@@ -7,6 +8,8 @@
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE TypeInType #-}
 {-# LANGUAGE EmptyCase #-}
+{-# LANGUAGE QuantifiedConstraints #-}
```

Comment:
Are you sure it needs `QuantifiedConstraints`? I don't recall it needing that when I tried writing this instance.

Reply:
Good catch, this is indeed not required.